lib/flutter_camera_ml_vision.dart

```diff
@@ -269,11 +277,11 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>> {
     );
   }
 
-  _processImage(CameraImage cameraImage) async {
+  void _processImage(CameraImage cameraImage) async {
```

Comment:
Future<void> instead no ?

Reply:
we pass this function to `camera` directly, and it doesn't care about the result, so void is fine